openlibrary/core/lists/model.py

```diff
@@ -162,6 +162,37 @@ def _get_edition_keys_from_solr(self, query_terms):
             for k in doc['edition_key']:
                 yield "/books/" + k
 
+    def get_export_list(self):
```

Comment:
```suggestion
    def get_export_list(self) -> list[dict]:
```
Python type hint for the return value?